AlphaWallet/Common/Views/TextField.swift

```diff
@@ -341,7 +341,7 @@ fileprivate extension TextField {
     @objc private func toggleMaskPassword(_ sender: UIButton) {
         isSecureTextEntry.toggle()
         if isSecureTextEntry {
-            sender.tintColor = Colors.navigationTitleColor
+            sender.tintColor = Configuration.Color.Semantic.navigationbarPrimaryFont
```

Comment:
Capital B in `navigationbarPrimaryFont`? Also seems like the semantic name is wrong. It's a UIButton right?

Reply:
Change this to `.secureIcon`?